machine.go

```diff
@@ -130,6 +129,17 @@ LinkLocalAddressing=no
 IPv6AcceptRA=no
 `
 const commandWrapper = `#!/bin/sh
+/lib/systemd/systemd-networkd-wait-online -q
+if [ $? != 0 ]; then
+  echo "WARNING: Network setup failed"
+  echo "== Journal =="
+  journalctl -a --no-pager
+  echo "== networkd =="
+  networkctl status
+  networkctl list
+  echo 1 > /run/fakemachine/result
+  exit
```

Comment:
just exit? without errorcode?

Reply:
Yeah the exit code isn't pass through to the fakemachine running on the host just the result file